.gitignore

```diff
@@ -1,16 +1,19 @@
-# build artifacts
-/dist
-/site/.hugo_build.lock
-/site/data/PluginGoDocs.json
-/site/data/PluginManifestDocs.json
-/site/data/PluginJSDocs.json
-/scripts/node_modules
-/scripts/mattermost-webapp
-/scripts/package-lock.json
-
-# os artifacts
-*.swp
-.DS_Store
-
-# IDE artifacts
-.idea/
+# build artifacts
+/dist
+/site/.hugo_build.lock
+/site/data/PluginGoDocs.json
+/site/data/PluginManifestDocs.json
+/site/data/PluginJSDocs.json
+/scripts/node_modules

```

Comment:
This line can be removed if there will no longer be a `node_modules` subdirectory in `scripts`.

Reply:
I have removed it from the `.gitignore` file.